### DISCUSSION TITLE
Pyinstaller for Tensorflow 2.11 GPU support for windows using WSL

### DISCUSSION THREAD

**Discussion by ps-raghotham-rao** [pyinstaller/pyinstaller#7422] [PyInstaller]
Hi, I have been trying to create an exe that is compatible with windows using WSL. As I cannot run tensorflow 2.11 gpu in windows so tried to create an env in WSL machine and creating an exe in that machine. But exe is not getting created but some linux based installer is getting created. Is there any approach to create exe in WSL machine?

**Comment by bwoodsend** [comment]
No, WSL is Linux. PyInstaller isn't going to magically translate Linux .so libraries into Windows DLLs.
